MD/md_gustorn.cpp

```diff
@@ -0,0 +1,194 @@
+﻿/*------------MD.cpp----------------------------------------------------------//
+*
+*             MD.cpp -- a simple event-driven MD simulation
+*
+* Purpose: Figure out the pressure on the interior of a box based on
+*          Boltzmann's theory and stuff
+*
+*   Notes: This simulation will predominantly follow the logic of the link in
+*          the README file
+*
+*-----------------------------------------------------------------------------*/
+
+#include <algorithm>
+#include <cstdint>
+#include <fstream>
+#include <iostream>
+#include <random>
+#include <vector>
+
+#include "math.h"
+
+namespace sim {
+    /* putting our stuff in its own namespace so we can call "time" "time"
+    without colliding with names from the standard library */
+    using time = double;
+
+/*----------------------------------------------------------------------------//
+* STRUCTS AND FUNCTIONS
+*-----------------------------------------------------------------------------*/
+
+// Holds our data in a central struct, to be called mainly in a vector
+struct Particle {
+    Point pos;
+    Vector vel;
+    uint64_t pid;
```

Comment:
`uint64_t` is better for indexes, you don't need negative numbers so this adds support for basicly an arbitrary number of  particles.